.meta/sinks/datadog_logs.toml

```diff
@@ -0,0 +1,46 @@
+[sinks.datadog_logs]
+title = "Datadog Logs"
+beta = true
+common = false
+delivery_guarantee = "best_effort"
+egress_method = "streaming"
+function_category = "transmit"
+healthcheck = true
+input_types = ["log"]
+service_providers = ["Datadog"]
+write_to_description = "[Datadog's][urls.datadog] logs"
+requirements = {}
+
+<%= render("_partials/_component_options.toml", type: "sink", name: "datadog_logs") %>
+
+<%= render("_partials/_buffer_options.toml", namespace: "sinks.datadog_logs.options") %>
+
+[sinks.datadog_logs.options.api_key]
+type = "string"
+common = true
+examples = ["${DATADOG_API_KEY_ENV_VAR}", "ef8d5de700e7989468166c40fc8a0ccd"]
+required = true
+description = "Datadog [API key](https://docs.datadoghq.com/api/?lang=bash#authentication)"
+
+[sinks.datadog_logs.options.encoding]
```

Comment:
Shouldn't this default to `json`?

Reply:
Since #1915 we don't default when there is more than one option, based on @binarylogic 's advice.